### PR TITLE
Fix return link for news releases

### PR DESCRIPTION
**Pull request by brangi**
## Description
Issue: https://github.com/department-of-veterans-affairs/va.gov-team/issues/10028

## Testing done
Locally

## Screenshots
ex

## Acceptance criteria
- [x] Create consistent user experience across Events, News Releases and Stories

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by ethanteague — APPROVED** [review]

File: src/site/layouts/press_release.drupal.liquid [hunk 1]
```diff
@@ -238,7 +238,7 @@
                         {{ fieldOffice.entity.fieldPressReleaseBlurb.processed }}
                     </section>
                     {% assign index = entityUrl.breadcrumb.length | minus: 2 %}
-                    <a onClick="recordEvent({ event: 'nav-secondary-button-click' });" href="{{ entityUrl.breadcrumb | getValueFromArrayObjPath: index, 'url.path' }}">See all news releases</a>
+                    <a onClick="recordEvent({ event: 'nav-secondary-button-click' });" href="{{ entityUrl.breadcrumb | getValueFromArrayObjPath: 2, 'url.path' }}">See all news releases</a>
```
Comment (mpelzsherman) [inline src/site/layouts/press_release.drupal.liquid:241]: This works, but it assumes that the breadcrumb element we want - the last one - is always at index 2. What if the breadcrumb length changes?

Wouldn't it be safer to change line 240 to 
```
{% assign index = entityUrl.breadcrumb.length | minus: 2 %}
```
??

Also, please note that there's a block of code that only runs in production and contains the same bug, starting on line 102. We really should go through the code and remove any prod-specific code that we can.

Comment (brangi) [reply src/site/layouts/press_release.drupal.liquid]: > This works, but it assumes that the breadcrumb element we want - the last one - is always at index 2. What if the breadcrumb length changes?
> 
> Wouldn't it be safer to change line 240 instead to
> 
> ??

Nice catch and yes this is better!

> 
> Also, please note that there's a block of code that only runs in production and contains the same bug, starting on line 102. We really should go through the code and remove any prod-specific code that we can.

Yes let's check with the team and fix these but in another issue

File: src/site/layouts/press_release.drupal.liquid [hunk 2]
```diff
@@ -237,7 +237,7 @@
                     <section class="vads-u-margin-bottom--3">
                         {{ fieldOffice.entity.fieldPressReleaseBlurb.processed }}
                     </section>
-                    {% assign index = entityUrl.breadcrumb.length | minus: 2 %}
+                    {% assign index = entityUrl.breadcrumb.length | minus: 1 %}
                     <a onClick="recordEvent({ event: 'nav-secondary-button-click' });" href="{{ entityUrl.breadcrumb | getValueFromArrayObjPath: index, 'url.path' }}">See all news releases</a>
```
Comment (mpelzsherman) [inline src/site/layouts/press_release.drupal.liquid:241]: 👍 

Shouldn't we apply the same fix to line 170?

**Comment by mpelzsherman** [comment]
Is there an e2e test for this link? Should we add one?

**Comment by brangi** [comment]
> Is there an e2e test for this link? Should we add one?

We'll add a lot of testing next sprint

**Review by mpelzsherman — APPROVED** [review]

**Review by bkjohnson — APPROVED** [review]

**Pull request merged**